### ISSUE TITLE
Whether training code can be exposed

### ISSUE THREAD

**Issue by jiaweimmiao** [levifussell/forging_new_worlds#1]
I was very interested in your paper "Forging new worlds: high-resolution synthetic galaxies with chained generative adversarial networks". I saw on github that you mentioned that you will be posting soon to train models from scratch using our own galaxy dataset. I did not find this model code for training from scratch, and I would like to ask if you can share your training code publicly.

**Comment by levifussell** [comment]
Hey @mjw1997, thanks for the message. This project is 4+ years old now, the training code exists but I'd have to dig it up. It's unlikely I will have the time for that anytime soon. But, I am happy to try to answer any direct questions about the training process you might be trying to answer by looking at said code.

**Comment by jiaweimmiao** [comment]
Thank you very much for your answer. I want to know how the loss function is defined in the training process. The definition of the loss function is not found in the paper, which is very difficult to reproduce the code. Look forward to your sharing.

**Comment by jiaweimmiao** [comment]
Sorry I read the paper again and saw the definition of the loss function in the paper, but I would like to see how you define, if possible, the initialization of the optimizer function for the discriminator and generator, and the learning rate for the generator and discriminator.
